docs/next/modules/en/pages/user/clusterclass.adoc

```diff
@@ -196,7 +254,7 @@ Azure RKE2::
 +
 [source,bash]
 ----
-kubectl apply -f https://raw.githubusercontent.com/rancher/turtles/refs/heads/main/examples/clusterclasses/azure/clusterclass-rke2-example.yaml
+kubectl apply -f https://raw.githubusercontent.com/rancher/turtles/refs/heads/main/examples/clusterclasses/azure/rke2/clusterclass-rke2-example.yaml
```

Comment:
I am getting not found error here and in all other similar changes, is it a typo or something?

Reply:
nvm, https://github.com/rancher/turtles/pull/1319 needs to be merged first I guess